### ISSUE TITLE
Need email notifications for results of the tests that are run on the PRs

### ISSUE THREAD

**Issue by itisravi** [gluster/project-infrastructure#91]
Just like in the gerrit work flow, it would be good to have an email notification of the results of the various tests that are run on a PR.  Currently, we need to go to the PR and check the status.

**Comment by mscherer** [comment]
I do not think we can change that. We could send email with a custom hook I guess, but then, not everybody would want that, and it would quickly evolve in a full fledge  notification systems that would duplicate the one from github already. I can see how annoying it is to have a workflow change,but I do not have a good solution coming to my mind for now.

**Comment by itisravi** [comment]
Can this custom hook email be sent only to the person who submitted the PR instead of a full fledged notification to all project members? That would help cut down on the noise.

**Comment by mscherer** [comment]
So I am looking at deploying a custom hook to send email for that repo (so we can get it sent on gluster-infra), but once I have that working, I can try to see for a failure notification hooks, if github publish the event.
